sc_preprocessing/0_qc_notebooks/scanpy_analysis_synovium_bulk.ipynb

```diff
@@ -0,0 +1,2142 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    del adata.raw
</pre><p>you're deleting the Unnormalized adata?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/buddi_analysis/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>